build-logic/build-update-utils/src/main/kotlin/gradlebuild.update-versions.gradle.kts

```diff
@@ -42,4 +43,15 @@ tasks.register<UpdateKotlinVersions>("updateKotlinVersions") {
     propertiesFile = layout.projectDirectory.file("gradle/dependency-management/kotlin-versions.properties")
 }
 
+tasks.register<UpdateKotlinEmbeddedVersion>("updateKotlinEmbeddedVersion") {
+    description = "Updates the embedded Kotlin version and the Kotlin version used in samples and snippets. " +
+        "When no -PnextVersion is passed to a task, latest version is fetched from MavenCentral."
+    rootDir = rootProject.layout.projectDirectory
+    nextVersion = providers.gradleProperty("nextVersion")
```

Comment:
:question: Why not use a task option instead of a Gradle property?

Reply:
Unfortunately, I wasn't able to do that today, but I can do it after the vacations